src/gui/mod.rs

```diff
@@ -764,7 +772,7 @@ impl Default for HeaderState {
                 ColumnState {
                     key: ColumnKey::LocalVersion,
                     btn_state: Default::default(),
-                    width: Length::Units(150),
+                    width: Length::Fill,
```

Comment:
Why are we setting this as `Fill`? Shouldn't we only have one as Fill, everything else is either the default value or whatever value is the result after a resize operation?